### PR TITLE
Replace templates "key = value" erb by unique template epp

### PR DESCRIPTION
**Pull request by phaedriel**
#### Pull Request (PR) description

- Remove 3 templates erb (key = value)
- Add 1 template epp (key = value)

#### This Pull Request (PR) fixes the following issues

- No one

### CODE REVIEW

File: templates/keyval.epp [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<% $kvlist.keys.sort.each | $k | { -%>
```
Comment (bastelfreak) [inline templates/keyval.epp:1]: please add a header to the template where it accepts the passed in parameters.

Comment (phaedriel) [reply templates/keyval.epp:1]: How could i add a header and not change file content (and service restart) ?

Comment (smortex) [reply templates/keyval.epp:1]: @phaedriel see https://www.puppet.com/docs/puppet/7/lang_template_epp.html#epp-parameter-tags

On a side note I saw a lot of closed PR / opened PR.  You do not need to open a new PR when you want to make some change: PRs are "linked" to your branch, so if you push new commit or rewrite history and force push, the PR is updating accordingly :wink:

Comment (phaedriel) [reply templates/keyval.epp:2]: Thanks !

I try merge master on my branch and rebase but my push contained "modulesync modification". Probably don't use correct git commands. Sorry

**Pull request closed without merging**